themes/harper/layouts/partials/header.html

```
@@ -0,0 +1,24 @@
+        {{ if eq .URL "/" }}
+          {{ $active = $cur.IsHome }}
+        {{ else }}
+          {{ $active = hasPrefix $cur.RelPermalink .URL }}
+        {{ end }}
+        <a href="{{ .URL | relURL }}" class="nav__link{{ if $active }} is-active{{ end }}">{{ .Name }}</a>
+      {{ end }}
+      {{ partial "language-switcher.html" . }}
+      {{ with .OutputFormats.Get "rss" }}
+        <a href="{{ .RelPermalink }}" class="nav__link nav__link--rss">RSS</a>
+      {{ else }}
+        <a href="{{ "index.xml" | relURL }}" class="nav__link nav__link--rss">RSS</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)